packages/sn-filter-pane/src/hooks/types/index.d.ts

```diff
@@ -44,7 +44,7 @@ export interface IListboxResource {
   id: string;
   model: GenericObjectModel;
   layout: IListLayout;
-  properties: EngineAPI.IGenericObjectProperties;
+  properties?: EngineAPI.IGenericObjectProperties;
```

Comment:
```suggestion
```
remove (this no longer has any properties)

Reply:
Updated it!